Maddori.Apple/Maddori.Apple/Screen/Home/Home/HomeViewController.swift

```diff
@@ -107,6 +113,18 @@ final class HomeViewController: BaseViewController {
         render()
     }
     
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        
+        // FIXME: - 테스트용으로는 넣어둔 데이터 -> UserDefaults로
+        fetchCertainTeamDetail(type: .fetchCertainTeamDetail(teamId: 40, userId: 83))
+        fetchCurrentReflectionDetail(type: .fetchCurrentReflectionDetail(teamId: 40, userId: 83))
```

Comment:
p2;

```suggestion
        fetchCertainTeamDetail(type: .fetchCertainTeamDetail(teamId: UserDefaultStorage.teamId, userId: UserDefaultStorage.userId))
        fetchCurrentReflectionDetail(type: .fetchCurrentReflectionDetail(teamId: UserDefaultStorage.teamId, userId: UserDefaultStorage.userId))
```
요기 머지하기 직전에 위와같이 변경해주세요 !

Reply:
[8d4641f](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/116/commits/8d4641fa4855ba404defb73b9ae01fc425b86d45)
!!